### DISCUSSION TITLE
zwave door lock schedules support

### DISCUSSION THREAD

**Discussion by priyanka043** [zwave-js/zwave-js-ui#3201] [Q&A]
Hi, 

I have added zwave kwikset door lock, i'm trying to add the schedules, in UI i'm unable to see regarding schedules to set weekday or yearday schedules, but in configurations i can see that the user code is owner or guest or worker, can anyone help me to add schedules, it would be the great help. 
![Screenshot (13)](https://github.com/zwave-js/zwave-js-ui/assets/139616028/1cf69c88-08fa-40f8-bedd-4d346503c79c)
![Screenshot (14)](https://github.com/zwave-js/zwave-js-ui/assets/139616028/38e0a9fb-5884-4e61-bc89-8b0844c32ecc)

**Comment by priyanka043** [comment]
Hi,

i'm able to add the schedules, but not working, i think due to time mismatch, can anyone help me regarding how to set time to the lock
![Screenshot (27)](https://github.com/zwave-js/zwave-js-ui/assets/139616028/5b5f0271-46c1-4dc3-b1a9-b1ecea3da7e9)
![Screenshot (28)](https://github.com/zwave-js/zwave-js-ui/assets/139616028/410eb6c9-bee8-440e-9ec8-39a336b6f628)

**Reply by robertsLando** [reply]
Click on node Advanced button > Sync time

**Comment by priyanka043** [comment]
but i want to manually set through date and time in time parameter value, how can i do from there, i mean the format.

**Comment by priyanka043** [comment]
I have kept schedules for weekday schedules, i have selected weekday monday whole day like 1 am to 11 pm, but the schedules where not working, Do i need to set some time or any other configurations to test the schedules, please help me with this schedules creation.

**Comment by robertsLando** [comment]
Some fixes for schedules are coming with: https://github.com/zwave-js/zwave-js-ui/pull/3176

Wait for a new release 🙏🏼

**Comment by priyanka043** [comment]
Thanks for your response, can you please let us know like till when this release will be released.

**Reply by robertsLando** [reply]
May be today or next week

**Comment by priyanka043** [comment]
thank you

**Comment by priyanka043** [comment]
Hi,

Just required an information does this #3176 schedules issues resolved and released?

**Reply by robertsLando** [reply]
I'm releasing it now

**Comment by priyanka043** [comment]
Thanks,

Is there any possibility in zwave door lock to delete all keys instead of deleting individually

**Reply by robertsLando** [reply]
Nope, only way would be using a custom driver function right now

**Comment by priyanka043** [comment]
Hi, 
I have downloaded the latest version of zwaveJS ui and i  tried to test the zwave door lock schedules, after creating schedule, while i try to delete that schedule i can see that i'm getting response as slot removed but actually even after deletion that schedule is still existing.

**Reply by robertsLando** [reply]
Is it the 8.22.3?

**Comment by priyanka043** [comment]
even when i try to update the existed schedule, it is not updating. can you please help regarding this

**Reply by robertsLando** [reply]
Please attach logs

**Comment by priyanka043** [comment]
yes

**Comment by priyanka043** [comment]
yes sure, PFA logs

**Reply by robertsLando** [reply]
? I don't see them
